pymeasure/instruments/tektronix/afg3152c.py

```diff
@@ -114,6 +120,22 @@ class Channel:
         validator=strict_range,
         values=FREQ_LIMIT,
     )
+    
+    phase_rad = Instrument.control( 
+        "phase:adjust?", "phase:adjust %e RAD",
+        """ A floating point property that controls the phase in radian units.
+        This property can be set.""",
+        validator = strict_range,
+        values = PHASE_LIMIT['RAD']
+    )
+    
+    phase_deg = Instrument.control(
+        "phase:adjust?", "phase:adjust %e DEG",
+        """ A floating point property that controls the phase in radian units.
+        This property can be set.""",
+        validator=strict_range,
+        values = PHASE_LIMIT['DEG']
```

Comment:
Probably a simple c&p mistake, for this property the text should be changed to `"...  in degrees."`

Reply:
Oops yea it was a copy and paste mistake.  And the string is actually different for phase_rad and phase_deg.  phase_deg ends with %e DEG, while phase_rad ends with %e RAD.

Reply:
no, the _query_ string, meaning the `get_command` argument is the same -- what's up with that? what happens if you do 
```
print(instr.phase_deg)
print(instr.phase_rad)
```
?

Reply:
It looks like phase_deg won't return the phase in degrees unless it's been set previously to degrees.  This also seems to be an issue with the get commands of amp_vpp and amp_vrms (haven't tried amp_dbm).  My AFG is being used in an ongoing measurement right now, but I wonder if just added the units RAD/DEG like so:

'phase:adjust RAD?'
'phase:adjust DEG?' 

would give the query back in the desired units.

Reply:
also apologies for the late replies, I've been working on my final required graduate course for my degree program.

Reply:
Good to have this confirmed. Such statefulness in instruments/command sets is always annoying.
To answer your question, I guess you need to study the manual and/or try it out when the instrument is free again.